pages/about.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Architectural Summit | About</title>
+    <link rel="stylesheet" href="../css/style.css" />
+  </head>
+  <body>
+    <header>
+      <ul class="social">
+        <li>
+          <a href="http://web.facebook.com/lawrence.amoafo"
+            ><ion-icon name="logo-facebook" id="icon"></ion-icon
+          ></a>
+        </li>
+        <li>
+          <a href="http://twitter.com/lawrence_amoafo"
+            ><ion-icon name="logo-twitter" id="icon"></ion-icon
+          ></a>
+        </li>
+        <li>English</li>
+        <li>My Page</li>
+        <li>Logout</li>
+      </ul>
+
+      <nav class="head">
+        <div class="vector">
+          <span></span>
+          <span></span>
+          <span></span>
+        </div>
+
+        <div class="logo">
+          <a href="../index.html"
+            ><img src="../Images/logo.png" alt="logo"
+          /></a>
+          <a href="../index.html"><b>World Architectural Summit 2023</b></a>
+        </div>
+
+        <ul class="nav-links">
+          <li><a href="../index.html">Home</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#location">Location</a></li>
+          <li><a href="#archives">Archives</a></li>
+        </ul>
+
+        <ul class="nav-links2">
+          <li><a href="../index.html">Home</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#location">Location</a></li>
+          <li><a href="#archives">Archives</a></li>
+          <li class="style">WA Summit</li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <section class="about">
+        <h1 class="about-heading">WORLD ARCHITECTURAL SUMMIT 2023</h1>
+        <p class="about-paragraph">
+          World Architectural Summit will build on the breakout success of the
+          inaugural edition with a captivating and insightful agenda that will
+          be discussed and debated by some of the region's best and brightest
+          minds. The Summit will provide a platform for construction industry
+          stakeholders to discuss a multitude of topics including future city
+          requirements, net zero building design, resilient design strategies,
+          future transport infrastructure, nurturing future talent and much
+          more. Along with a rich agenda, the summit will also provide
+          unparalleled networking opportunities in full compliance with all
+          local COVID-19 related regulations.
+        </p>
+      </section>
```

Comment:
- [ ] Here in this section the about-paragraph has extra top margin. kindly consider decreasing the gap between the heading and the paragraph as in the Behance design because our design needs to be similar to the Behance.
- [ ] There must be a text line under `about-paragraph` containing the contact information as in the original design in Behance.
your design
![fd](https://user-images.githubusercontent.com/96683843/225998769-d94fceee-9f34-4175-a755-f19791ddc7d5.jpg)

original design
![as](https://user-images.githubusercontent.com/96683843/225998750-7f120d24-4138-46fd-9c29-71c9ffc160c2.jpg)